backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -742,4 +742,8 @@ SearchException invalidSingleValuedProjectionOnValueFieldInMultiValuedObjectFiel
 	@Message(id = ID_OFFSET + 159, value = "Invalid use of 'missing().lowest()' for a descending distance sort. " +
 			"Elasticsearch always assumes missing values have a distance of '+Infinity', and this behavior cannot be customized. ")
 	SearchException missingLowestOnDescSortNotSupported(@Param EventContext context);
+
+	@Message(id = ID_OFFSET + 160, value = "'%1$s' projections cannot be nested in an object projection. "
```

Comment:
The word "projection" already appears in the projection key.

```suggestion
	@Message(id = ID_OFFSET + 160, value = "'%1$s' cannot be nested in an object projection. "
```